### DISCUSSION TITLE
Unable to disable root_folder_label

### DISCUSSION THREAD

**Discussion by Yohnny777** [nvim-tree/nvim-tree.lua#1909] [Q&A]
### Description

Setting `renderer.root_folder_label` to an empty string or any string doesn't show it but instead falls back to full path like "C:\Users\user\project`. 

Setting it to ':h' also falls back to full absolute path instead of short.

### Neovim version

```text
0.8.2
```

### Operating system and version

Windows 10, Terminal

### nvim-tree version

master

### Minimal config

```lua
any
```

### Steps to reproduce

1. set `renderer.root_folder_label` to "", "My label" or ":h"

### Expected behavior

Label is equal to the string or just directory name for ":h" instead of entire absolute path

### Actual behavior

its always absolute path

**Comment by alex-courtis** [comment]
`renderer.root_folder_label` must be a vaild `:help filename-modifiers`.

`:h` gives you the head i.e. cwd name and is probably not useful. `:t` is what I use to show just the cwd name.

If you like, you can customise this label further via a function `:help nvim-tree.renderer.root_folder_label`

You can disable it completely via `:help nvim-tree.view.hide_root_folder`.
